test/index-removeItems.test.js

```diff
@@ -1,29 +1,29 @@
 const mockAWSSinon = require("mock-aws-sinon");
 
-describe("Testing backoff", function () {
-	this.timeout(50000);
+const SimpleDynamo = require("../index");
 
-	describe("base test", function () {
-		const SimpleDynamo = require("../index");
-		const database = new SimpleDynamo(
-			{},
-			{
-				region: "eu-west-1",
-				dynamodb: "2012-08-10"
-			}
-		);
+describe("Testing backoff", () => {
+	jest.setTimeout(50000);
 
-		it("Remove should return UnprocessedItems", function (done) {
-			let i = 0;
-			const items = [];
+	describe("base test", () => {
+		let i = 0;
+		const items = [];
 
-			while (i < 200) {
-				items.push({
-					something: "cool"
-				});
+		while (i < 200) {
+			items.push({
+				something: "cool"
+			});
+			i++;
+		}
 
-				i++;
-			}
+		it("Remove should return UnprocessedItems", async () => {
+			const database = new SimpleDynamo(
+				{},
+				{
+					region: "eu-west-1",
+					dynamodb: "2012-08-10"
+				}
+			);
 
 			mockAWSSinon("DynamoDB", "batchWriteItem")
```

Comment:
Isn't `sinon` used for creating Spies? [Mocks are preferred over spies](https://cloudconformity.atlassian.net/wiki/spaces/IN/pages/719585287/Unit+testing#Mocks-over-Spies)?

Reply:
Sinon also has Stubs and Mocks which is what this library does. I can assure you that this does actually does mock instead of spy as it runs without assuming role.